openedx/adg/lms/applications/views.py

```diff
@@ -97,6 +97,7 @@ def get(self, request):
             HttpResponse object.
         """
         user_application_hub = ApplicationHub.objects.get(user=self.request.user)
+        user_application = UserApplication.objects.get(user=self.request.user)
```

Comment:
For the new user this will break as there is no application at the start. Pls handle accordingly.
<img width="1163" alt="Screenshot 2021-08-23 at 7 23 43 PM" src="https://user-images.githubusercontent.com/52656433/130464160-9daf0ff2-81ef-4123-ba9b-5f7259dd89ac.png">